core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/AutoscaleV4Controller.java

```diff
@@ -132,10 +158,37 @@ public AutoscaleStackV4Response getInternalAutoscaleClusterByName(String name, @
     @InternalOnly
     public void decommissionInternalInstancesForClusterCrn(@TenantAwareParam @ResourceCrn String clusterCrn,
             List<String> instanceIds, Boolean forced) {
+        LOGGER.info("decommissionInternalInstancesForClusterCrn. forced={}, clusterCrn={}, instanceIds=[{}]",
+                forced, clusterCrn, instanceIds);
         stackCommonService.deleteMultipleInstancesInWorkspace(NameOrCrn.ofCrn(clusterCrn), restRequestThreadLocalService.getRequestedWorkspaceId(),
                 new HashSet(instanceIds), forced);
     }
 
+    @Override
+    @InternalOnly
+    public void stopInternalInstancesForClusterCrn(@TenantAwareParam @ResourceCrn String clusterCrn, @NotEmpty List<String> instanceIds,
```

Comment:
start API is available for user, why this internal?

Reply:
start/stop instances will initially only be used by AutoScale/Periscope. Eventually, we'll introduce a CLI to support this, at which point they'll need to be public.
Will check about the entitlement and add it in.

Reply:
thank you!

Reply:
Fixed in the next push. Resolving conversation.